integrations/logging/loki/uninstall.sh

```diff
@@ -2,7 +2,9 @@
 # Copyright KubeArchive Authors
 # SPDX-License-Identifier: Apache-2.0
 NAMESPACE="grafana-loki"
+VECTOR_NAMESPACE="kubearchive-vector"
 
 kubectl delete namespace ${NAMESPACE}
+kubectl delete namespace ${VECTOR_NAMESPACE}
```

Comment:
Taking a second look to this, is there a reason to use a different namespace for vector? Could we use `vector-grafana-loki` for the 3 components?

Reply:
ok!

Nevertheless, we should include a flag for the script, and in case the flag is provided conditionally remove the logging-operator stuff (if we install loki with vector there is no logging-operator so the uninstall script will fail)

```
for i in "$@"
do
case $i in
    --vector)
    VECTOR=True
    ;;
    *)
    echo "Unknown option $i" # unknown option
    HELP=True
    UNKNOWN=True
    ;;
esac
done

VECTOR=${VECTOR:-"False"}

kubectl delete namespace ${NAMESPACE}

if [ "${VECTOR}" == "False" ]; then
    kubectl delete clusterrole logging-operator logging-operator-edit
    kubectl delete clusterrolebinding logging-operator
fi
```